workloads/upgrade-perf/README.md

```diff
@@ -15,11 +15,20 @@ $ ./run_<test-name>_fromgit.sh
 Default: ''
 The target version of the cluster. THIS IS A REQUIRED VARIABLE
 
-## TOIMAGE
+### TOIMAGE
 Default: ''
 This is an optional location of an image to upgrade to. If set you will STILL need to assign
 the VERSION variable as well to match what the end result will be.
 
+### LATEST
+Default: ''
+Upgrades to the latest version available in the channel when set. If set you will STILL need to 
+assign the VERSION variable as well to match what the end result will be.
```

Comment:
How I have to set LATEST if I want to use it? `export LATEST=true`?

Reply:
Right

Reply:
Modified the README to set it to true in case of upgrading to the latest available build.